Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+description: Chart for Timed application
+name: timed
+version: 0.1.0
+keywords:
+  - timed
+  - tracking
+sources:
+  - https://github.com/adfinis-sygroup/timed-chart
```

Comment:
I would also add a link to the timed source code here.

Reply:
Make sense. Added.